test/FunctionsV1/PlatformSpecificHelpers.FunctionsV1.cs

```diff
@@ -37,6 +37,7 @@ public static class PlatformSpecificHelpers
 #pragma warning restore CS0612 // Type or member is obsolete
         {
             var config = new JobHostConfiguration { HostId = "durable-task-host" };
+            // var config = new JobHostConfiguration("UseDevelopmentStorage=true;DevelopmentStorageProxyUri=http://127.0.0.1:10002/");
```

Comment:
I'm guessing this was temporary code for local testing? Can it be deleted?

Reply:
Done